lib/jsonapi/serializer.ex

```diff
@@ -135,11 +135,27 @@ defmodule JSONAPI.Serializer do
     merge_related_links(data, info, remove_links?())
   end
 
-  defp merge_links(doc, data, view, conn, false) do
+  defp merge_links(doc, data, view, conn, false, true) do
+    pagination_links = view.links(data, conn)
+
+    if Enum.empty?(pagination_links) do
+      IO.warn(
```

Comment:
Shouldn't we use Logger?

Reply:
I'm fine with that. I was not sure what would be the best way to pass this along to the user. What log level though? info?